src/server.js

```diff
@@ -70,7 +70,8 @@ function createRemixRequest(request, options = {}) {
     headers: request.headers,
     signal: controller.signal,
     // eslint-disable-next-line unicorn/no-null -- Request init expects a `null` value.
-    body: isGetOrHead(request) ? request.body : null,
+    body: isGetOrHead(request) ? null : request.clone().body,
+    duplex: request.method !== 'GET' && request.method !== 'HEAD' ? 'half' : undefined,
```

Comment:
Wouldn't the suggestion be the same?

```suggestion
    duplex: isGetOrHead(request) ? undefined : 'half',
```